### ISSUE TITLE
File naming for transparent images should ignore querystring

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#209]
```
The current implementation of the library doesn't fix images that are named 
-trans.png but contain 
a query-string. All my assets in my web application are automatically suffixed 
with a version 
number to ensure the right version is used (not a wrong one from the cache). 
Unfortunately the ie7 
library doesn't fix those images..
This issue occurs with the beta3
```

Original issue reported on code.google.com by `marijn.huizendveld@gmail.com` on 16 Nov 2009 at 12:26

**Comment by GoogleCodeExporter** [comment]
```
This is a duplicate of Issue #214.

It is fixed in the next release (2.1).
```

Original comment by `dean.edw...@gmail.com` on 10 Feb 2010 at 9:28
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
Nice, sorry for filing a duplicate ticket, I guess I didn't search well enough 
:-$
```

Original comment by `marijn.huizendveld@gmail.com` on 10 Feb 2010 at 9:32
